### PR TITLE
add anomly detector notebook for zouwu

### PR DESCRIPTION
**Pull request by EN1AC13**
Update:
- replace current anomaly detection method with anomaly detector API
- modify helper functions
- add some docs

### CODE REVIEW

File: pyzoo/zoo/zouwu/use-case/AIOps/AIOps_anomaly_detect.ipynb [hunk 1]
```diff
@@ -0,0 +1,1410 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "full_idx= pd.Index([i for i in range(resampled_df.index.min(),resampled_df.index.max()+1)])\n",
+    "resampled_df = resampled_df.reindex(full_idx)\n",
```
Comment (yushan111) [inline pyzoo/zoo/zouwu/use-case/AIOps/AIOps_anomaly_detect.ipynb:461]: I am a little confused about this part. What is it for?

File: pyzoo/zoo/zouwu/use-case/AIOps/AIOps_anomaly_detect.ipynb [hunk 2]
```diff
@@ -0,0 +1,1410 @@
...
+   "execution_count": 20,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "df = df.reset_index(drop=True)\n",
+    "df.index=pd.to_datetime(df.index, unit='m', origin=pd.Timestamp('2018-01-01'))"
```
Comment (yushan111) [inline pyzoo/zoo/zouwu/use-case/AIOps/AIOps_anomaly_detect.ipynb:680]: I think you have added datetime index before.

File: pyzoo/zoo/zouwu/use-case/AIOps/AIOps_anomaly_detect.ipynb [hunk 3]
```diff
@@ -0,0 +1,1410 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "use anomaly detector interface in zoo.zouwu.model.anomaly"
```
Comment (yushan111) [inline pyzoo/zoo/zouwu/use-case/AIOps/AIOps_anomaly_detect.ipynb:1204]: we can add another title like 'Anomaly detection' to make it more clear

**Comment by EN1AC13** [comment]
Ok.I will revise it.


Thanks,
Ruolan



发自我的iPhone


------------------ Original ------------------

File: pyzoo/zoo/zouwu/use-case/AIOps/AIOps_anomaly_detect.ipynb [hunk 4]
```diff
@@ -0,0 +1,1431 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "Last, we generate model input samples by sliding window along time axis. Since univariant and multivariant forcasting uses different input data shape, we prepare data for each type of forcasting respectively."
```
[the remark this answers is not in the document]
Comment (EN1AC13) [reply pyzoo/zoo/zouwu/use-case/AIOps/AIOps_anomaly_detect.ipynb:985]: I have fix typos in new PR.
The latest anomaly notebook is in new PR https://github.com/intel-analytics/analytics-zoo/pull/2374

**Comment by EN1AC13** [comment]
I have fix typos in new PR.
The latest anomaly notebook is in new PR #2374

**Comment by yushan111** [comment]
the updated version is in PR #2374

**Pull request closed without merging**